cypress/e2e/browse.cy.js

```diff
@@ -1,12 +1,14 @@
-describe('The home page', () => {
-  it('should be able to reach the browse page', () => {
+describe('Browse', () => {
```

Comment:
as I think about it more, because these are not unit tests, perhaps it's easier to think about the feature when considering the naming. not the component it starts/ends with.

I'll update the ticket to reflect this. lmk what you think.

Reply:
@jeremyf since you have strong opinions about cypress...what do you think about this "discussion"?

Reply:
I definitely like the idea of naming the feature.  At Forem, our Cypress tests would start with functional areas and drill down from there.  The other part that I really loved is that we used `aria-role` based selection instead of CSS selector or anything like that.  That meant we had a more accessible application because we asserted role behavior

Reply:
@jeremyf @alishaevn
ok - based on the new ticket requirements, I updated the test to correspond with 
" a user can search from the home page and be directed to "/browse" with the proper query" from the ticket

I also made a ticket to add aria-role throughout the app for a11y (and then update the tests as needed) since we had not been doing that